lib/octokit/client/licenses.rb

```diff
@@ -27,6 +27,19 @@ def license(license_name, options = {})
         options = ensure_api_media_type(:licenses, options)
         get "licenses/#{license_name}", options
       end
+
+      # Returns the contents of the repository’s license file, if one is detected.
+      #
+      # @see https://developer.github.com/v3/licenses/#get-a-repositorys-license
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @option options [String] :ref name of the Commit/Branch/Tag. Defaults to “master”.
+      # @return [Sawyer::Resource] The detail of the license file
+      # @example
+      #   Octokit.license_contents 'benbalter/licensee'
+      def license_contents(repo, options = {})
+        options = ensure_api_media_type(:licenses, options)
+        get "#{Repository.path repo}/license", options
```

Comment:
>>  If the content is Base64, can we decode that before returning to the caller?

> We can, the other content methods (e.g., README, doesn't as far as I can see).

I was actually thrown off by the other methods not doing this the other day, especially since you have to call `.force_encoding("UTF-8")` after decoding it.